sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -671,17 +671,25 @@ internal static void CheckNoCashPositionsInValuationResults(ListAggregationRespo
             Assert.That(doesNotContainAnyCashBeforeExpiration, Is.True);
         }
         
+        // TODO: requestForUnderlying returns instrument underlying, if any. 
+        // TODO: It is currently used for EquityOption physically-settled lifecycle test.
+        // TODO: In the long term, we should not need this and LUSID auto-adds underlying of
+        // TODO: the option into LUSID upon expiration.
         internal static ValuationRequest CreateValuationRequest(
             string scope,
             string portfolioCode,
             string recipeCode,
             DateTimeOffset effectiveAt,
-            DateTimeOffset? effectiveFrom = null)
+            DateTimeOffset? effectiveFrom = null,
+            bool requestForUnderlying = false) 
```

Comment:
Consider adding method comments that describe the parameters (particularly the new one)

Reply:
Already have on L674 -L677 :)
[only the bool is new]